server/Cargo.toml

```diff
@@ -1,21 +1,29 @@
 [package]
-name = "stackable-nifi-operator-server"
-version = "0.1.0-nightly"
 authors = ["Malte Sander <malte.sander.it@gmail.com>"]
+build = "build.rs"
 edition = "2018"
 description = "Stackable Operator for Apache NiFi"
 license = "OSL-3.0"
+name = "stackable-nifi-operator-server"
+repository = "https://github.com/stackabletech/nifi-operator"
+version = "0.1.0-nightly"
 
 [dependencies]
-stackable_config = { git = "https://github.com/stackabletech/common.git", branch = "main" }
+stackable-config = { git = "https://github.com/stackabletech/common.git", branch = "main" }
```

Comment:
```suggestion
stackable-config = { git = "https://github.com/stackabletech/common.git", tag = "0.1.0" }
```

Reply:
Its actually not even required. Ill remove it.

Reply:
Even better